Mesh_3/doc/Mesh_3/CGAL/Mesh_3/parameters.h

```diff
@@ -451,6 +451,72 @@ unspecified_type odt(const Named_function_parameters& np = parameters::default_v
+ * for the initialization of the meshing process. If this parameter is specified without arguments, the default behavior
+ * is executed, which calls the domain's `construct_initial_points_object()` for the initialization of the meshing process.
+ *
+ * If the parameter `parameters::initial_points()` is used, the points it provides are inserted before any other initialization.
+ *
+ * If the generator does not generate enough points, the domain's `construct_initial_points_object()` will be called.
+ *
+ * \tparam InitialPointsGenerator a functor that follows the `InitialPointsGenerator` concept
+ *
+ * @param generator an instance of `InitialPointsGenerator`
+ *
+ * \cgalHeading{Example}
+ *
+ * \snippet mesh_3D_image_with_image_initialization.cpp Meshing
+ *
+ * \sa `CGAL::make_mesh_3()`
+ * \sa `CGAL::parameters::initial_points()`
+ * \sa `MeshDomain_3::Construct_initial_points`
+ *
+ */
+template <typename InitialPointsGenerator>
+unspecified_type initial_points_generator(const InitialPointsGenerator& generator);
+
+/*!
+ * \ingroup PkgMesh3Parameters
+ *
+ * The function `parameters::initial_points()` enables the user to specify a container model of `Range` that contains
+ * initial points to be used in the `make_mesh_3()` function for mesh generation. The `Range` contains
+ * tuple-like objects containing a `Weighted_point_3`, an `int`, and an `Index`,
+ * where `Weighted_point_3` represents the position and weight of the point,
+ * `int` the dimension of the minimal subcomplex on which the point lies,
+ * and `Index` the corresponding subcomplex index.
+ *
+ * If the parameter `parameters::initial_points_generator()` is set, the points will be inserted before calling the functor.
+ * If the insertion of initial points, together with the input generator, do not generate enough points,
+ * the domain's `construct_initial_points_object()` will be called.
+ *
+ * \tparam MeshDomain a model of `MeshDomain_3`
+ * \tparam C3t3 a model of `MeshComplex_3InTriangulation_3`
+ *
+ * @param initial_points a `Range` that contains points of tuple-like objects of
+ *  `C3t3::Triangulation::Geom_traits::Weighted_point_3, int, MeshDomain::Index`.
+ *
+ * \cgalHeading{Example}
+ *
+ * \code{.cpp}
+ * // Create the initial_points vector
+ * std::vector<std::tuple<K::Weighted_point_3, int, Mesh_domain::Index>> initial_points;
+ * // Perform mesh generation from a labeled image with initial points
+ * C3t3 c3t3 = make_mesh_3<c3t3>(domain,
+ *                               criteria,
+ *                               parameters::initial_points(std::cref(initial_points))); // Use std::cref to avoid a copy
+ * \endcode
+ *
+ * \sa `CGAL::parameters::initial_points_generator()`
+ * \sa `CGAL::make_mesh_3()`
+ * \sa `MeshDomain_3::Construct_initial_points`
+ *
+ */
+template <typename MeshDomain, typename C3t3>
+unspecified_type initial_points(const std::vector<std::tuple<C3t3::Triangulation::Geom_traits::Weighted_point_3, int, MeshDomain::Index>>& initial_points);
```

Comment:
You have a vector hardcoded here instead of the `typename Range`  / instance of `Range` that you document